### PR TITLE
Move ArcGis map to ESModule syntax

### PR DESCRIPTION
**Pull request by BaxLi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +35,7 @@
     "@ngx-translate/http-loader": "^6.0.0",
     "bootstrap": "^4.5.3",
     "esri-loader": "^3.0.0",
+    "moment": "^2.29.1",
```
Comment (unrevised6419) [inline package.json]: I think we don't need this.

**Review by unrevised6419 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (BaxLi) [reply package.json]: yep - you are right.

Comment (BaxLi) [reply package.json]: blin am facut - dar in loc sa se adauga in comitul dat s-a facut comit nou ... sorry

Comment (unrevised6419) [reply package.json]: Am inchis acel PR care ai deschis gresit.

Comment (unrevised6419) [reply package.json]: @BaxLi can you remove it, so I can merge?

Comment (BaxLi) [reply package.json]: done

**Comment by unrevised6419** [comment]
@BaxLi do you get any warnings when you try to start the application? I see also that it has @arcgis/core has moment js as a dependency, which has a big payload.

<details>
<summary>Logs</summary>

```log
Warning: ./node_modules/@arcgis/core/core/workers/workers.js 5:863-872
Critical dependency: the request of a dependency is an expression

Warning: ./node_modules/@arcgis/core/core/workers/WorkerFallback.js 5:2264-2273
Critical dependency: the request of a dependency is an expression

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/zh-tw.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/zh-mo.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/zh-hk.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/zh-cn.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/vi.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/uk.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/tr.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/th.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/sv.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/sr-cyrl.js'. CommonJS or AMD dependencies can cause optimization bailouts.[[39m39m
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/sr.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/sl.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/sk.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ru.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ro.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

[0[39mmWarning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/pt-br.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/pt.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/pl.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/nl-be.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/nl.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/nb.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/lv.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/lt.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ko.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ja.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/it-ch.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/it.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/id.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/hu.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/hr.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/he.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/fr-ch.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/fr-ca.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/fr.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/fi.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/et.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/es-us.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/es-mx.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/es-do.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/es.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/en-sg.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/en-nz.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

[39m[0mWarning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/en-in.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/en-il.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/en-ie.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/en-gb.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/en-ca.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/en-au.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/el.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/de-ch.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/de-at.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/de.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/da.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/cs.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ca.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/bs.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ar-tn.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ar-sa.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ar-ma.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ar-ly.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ar-kw.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ar-dz.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/iamandrewluca/Projects/github.com/iamandrewluca/voluntar-web/node_modules/@arcgis/core/intl/moment.js depends on 'moment/locale/ar.js'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```
</details>

Maybe we should stick with `esri-loader` till `@arcgis/core` is in `beta` phase?

**Comment by unrevised6419** [comment]
I'm investigating the problem. Maybe will merge and ignore the warning.

**Comment by unrevised6419** [comment]
I'll close the PR for now. The build is failing with a lot of warnings, we will move to `@arcgis/core` when will be out of beta.
I tried multiple ways of loading them, but none of them is working, and our bundle size has doubled I think

**Pull request closed without merging**